test/utils/run_all_unit.sh

```diff
@@ -1,4 +1 @@
-for f in test/unit_tests/*.py
-do
-   bash -ex -c "cd test/unit_tests; python `basename $f`"
-done
+nosetests -v --with-doctest
```

Comment:
We probably don't need run_all_utils.sh anymore...

Reply:
I have removed the file.